.github/actions/deploy-ember-preview/action.yml

```diff
@@ -10,29 +10,34 @@ inputs:
     required: true
     default: web-client
 
-env:
-  VOLTA_FEATURE_PNPM: 1
-
 runs:
   using: composite
   steps:
     - uses: volta-cli/action@v4
+
+    - shell: bash
+      run: echo "VOLTA_FEATURE_PNPM=1" >> $GITHUB_ENV
+
     - name: Install Dependencies
       shell: bash
       run: pnpm install --frozen-lockfile
+
     - name: Set PR branch name
       shell: bash
       run: echo "RAW_PR_BRANCH_NAME=${{ github.head_ref }}" >> $GITHUB_ENV
+
     - name: Convert branch name to subdomain-compatible form
       shell: bash
       run: echo "PR_BRANCH_NAME=`echo ${RAW_PR_BRANCH_NAME} | tr _ - | tr '[:upper:]' '[:lower:]' | sed -e 's/-$//' | sed -e 's/[^a-z0-9\-]//g'`" >> $GITHUB_ENV
+
     - name: Deploy preview
       shell: bash
       run: pnpm deploy:${{ inputs.package }}:preview-${{ inputs.environment }}
 
     - name: Store preview host
       shell: bash
       run: echo "PREVIEW_HOST=https://$PR_BRANCH_NAME.$S3_PREVIEW_BUCKET_NAME/" >> $GITHUB_ENV
+
```

Comment:
I personally prefer these gaps but is this a lint thing?

Reply:
It's not a lint thing sadly. I was fixing the `VOLTA_FEATURE_PNPM` and added the gaps so that it's easier on the eyes.